WcaOnRails/lib/country_city_validators/gb_city_validator.rb

```diff
@@ -92,6 +92,17 @@
   Wiltshire
   Worcestershire
   Yorkshire
+  Akrotiri\ and\ Dhekelia
+  Anguilla
+  Bermuda
+  British\ Virgin\ Islands
+  Cayman\ Islands
+  Falkland\ Islands
+  Gibraltar
+  Montserrat
+  Pitcairn\ Islands
+  Saint\ Helena,\ Ascension\ and\ Tristan\ da\ Cunha
+  Turks\ and\ Caicos\ Islands
 ).to_set
 
 CityValidator.add_validator_for_country "GB", CityCommaRegionValidator.new(type_of_region: "county", valid_regions: GB_COUNTIES)
```

Comment:
Should we also change `type_of_region` to "county or overseas territory"? It would be more correct but is kinda 'in your face'.

Reply:
For simplicity, I left it as `county`, although I guess that a different type can be added later on and this be changed if you think it's necessary!